recipes/datasette/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "datasette" %}
+{% set version = "0.26.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3856bf31482428dd89e965ebd090b69a844bac814f807e3317c05a0c631dfb57
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - click
+    - click-default-group
+    - sanic
+    - jinja2
+    - hupper
+    - pint
+    - pluggy
+
+test:
+  commands:
+    - datasette --help
+
+about:
+  home: https://github.com/simonw/datasette
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
```

Comment:
The license is included in the sdist, so while you can leave this line untouched, please remove the `LICENSE` file you manually added to this feedstock.

Reply:
I am sorry, my bad. The license is _not_ included in the sdist. Sorry for the confusion.

Could you please open an issue or PR upstream to include the license in the source distribution?